secrethub.env

```diff
@@ -0,0 +1 @@
+HTTPBIN_BEARER = {{ $app/httpbin/bearer }}
```

Comment:
Can we call this `HTTPBIN_TOKEN` or something? This doesn't signal to me it's a secret. Only if you're familiar with the concept of a `bearer token` does this make some sense.